data-transformers/transformers/statsTransformer.py

```diff
@@ -46,20 +47,24 @@
 data = sc.parallelize(lines)
 
 # Creates a dictionary
-def createDict(a):
+def createEDDict(a):
     ob = json.loads(a.decode())
     d = {}
+    activeCpus = 0
+    for c in ob["cpu_stats"]["cpu_usage"]["percpu_usage"]:
```

Comment:
Do you really need to do it for each data point? Isn't enough to do it just once? Maybe towards supporting live resources updated happened during the container execution (this is not yet available in Docker, at least)?